docs/.vuepress/config.js

```diff
@@ -9,120 +9,138 @@ const PUBLISH_TARGET_PATH = (process.env.PUBLISH_TARGET_PATH || 'latest').replac
+      'user-guide/install-zos.md',
+      'user-guide/cli-installcli.md',
+      'user-guide/uninstall.md',
     ]
   },
   {
-    text: 'Extending',
-    baseuri: '/extend/',
+    text: 'Configuring Zowe',
     items: [
-      {
-        text: 'Developing for API Mediation Layer',
-        items: [
-          'extend/extend-apiml/api-mediation-onboard-overview.md',
-          'extend/extend-apiml/api-mediation-security.md',
-          'extend/extend-apiml/api-mediation-onboard-a-sprint-boot-rest-api-service.md',
-          'extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md',
-          'extend/extend-apiml/api-mediation-onboard-an-existing-java-jersey-rest-api-service.md',
-          'extend/extend-apiml/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.md',
-        ]
-      },
-      {
-        text: 'Developing for Zowe CLI',
-        items: [
-          'extend/extend-cli/cli-devTutorials.md',
-          'extend/extend-cli/cli-setting-up.md',
-          'extend/extend-cli/cli-installing-sample-plugin.md',
-          'extend/extend-cli/cli-extending-a-plugin.md',
-          'extend/extend-cli/cli-developing-a-plugin.md',
-          'extend/extend-cli/cli-implement-profiles.md',
-        ]
-      },
-      {
-        text: 'Developing for Zowe Application Framework',
-        items: [
-          'extend/extend-desktop/mvd-extendingzlux.md',
-          'extend/extend-desktop/mvd-creatingappplugins.md',
-          'extend/extend-desktop/mvd-plugindefandstruct.md',
-          'extend/extend-desktop/mvd-dataservices.md',
-          'extend/extend-desktop/mvd-desktopandwindowmgt.md',
-          'extend/extend-desktop/mvd-configdataservice.md',
-          'extend/extend-desktop/mvd-uribroker.md',
-          'extend/extend-desktop/mvd-apptoappcommunication.md',
-          'extend/extend-desktop/mvd-errorreportingui.md',
-          'extend/extend-desktop/mvd-logutility.md',
-          'extend/extend-desktop/zlux-app-server.md',
-          'extend/extend-desktop/zlux-workshop-user-browser.md',
-          'extend/extend-desktop/zlux-tutorials.md',
-          'extend/extend-desktop/starter-intro.md',
-          'extend/extend-desktop/zlux-workshop-starter-app.md',
-          'extend/extend-desktop/ui-intro.md',
-          'extend/extend-desktop/iframe-sample.md',
-          'extend/extend-desktop/angular-sample.md',
-        ]
-      }
+      'user-guide/mvd-configuration.md',
+      'user-guide/cli-configuringcli.md',
     ]
   },
   {
-    text: 'Troubleshooting',
-    baseuri: '/troubleshoot/',
-    link: 'troubleshoot/troubleshootinstall.md'
+    text: 'Using Zowe',
+    items: [
+      'user-guide/mvd-using.md',
+      'user-guide/api-mediation-api-catalog.md',
+      'user-guide/cli-usingcli.md',
+    ]
   },
   {
-    text: 'Contributing',
-    baseuri: '/contributing.html',
-    link: 'contributing.md',
-    canHideFirst: true,
```

Comment:
@BrandonJenkins14 you can keep the `canHideFirst: true,` on `Contributing` page. With this, both of Contributing and Appendix links will be hidden early than others.